servers/rendering/renderer_rd/renderer_scene_sky_rd.h

```diff
@@ -80,16 +93,52 @@ class RendererSceneSkyRD {
 		SKY_VERSION_MAX
 	};
 
-	// Skys need less info from Directional Lights than the normal shaders
-	struct SkyDirectionalLightData {
-		float direction[3];
-		float energy;
-		float color[3];
-		float size;
-		uint32_t enabled;
-		uint32_t pad[3];
+	struct SkyPushConstant {
+		float orientation[12]; // 48 - 48
+		float projections[RendererSceneRender::MAX_RENDER_VIEWS][4]; // 2 x 16 - 64, if we ever need more then 3 we should consider adding this to a set.
+		float position[3]; // 12 - 92
+		float multiplier; // 4 - 96
+		float time; // 4 - 100
+		float pad[3]; // 12 - 112
```

Comment:
```suggestion
		float projections[RendererSceneRender::MAX_RENDER_VIEWS][4]; // 2 x 16 - 80
		float position[3]; // 12 - 108
		float multiplier; // 4 - 112
		float time; // 4 - 116
		float pad[3]; // 12 - 128, 128 is the max size of a push constant. We can replace "pad" but we can't add any more.
```

Reply:
Cheers, changed this :)

Reply:
Okay, looks like my math was bad. I just blindly added 16 because the size for projections was wrong. But as it turns out, the other sizes were correct in the first place... 80 + 12 is not 108

Reply:
hahahaha, and here I thought I had a lapse of concentration, I just took your numbers as is :)

Reply:
Ok, so we need to pad by 3 to align by 16 bytes, so the push constant only needs to be 112 bytes, it doesn't need to be expanded to 128 bytes.